### PR TITLE
Add get random topic and its translations query

### LINKED ISSUE #26: Get random conversation topic + translations query
- [ ] Get random conversation topic that has user's native + target language translations

### PR DESCRIPTION
**Pull request by leahriffell**
### What was changed?
- **create method to get a random conversation topic and its translations based on the languages requested (currently client can *only* pass in two language ids)**
   - created ticket #81 to refactor this in the long run 
- New GQL types: Topic, TopicTranslation
- New factories: Topic, TopicTranslation
- New Topic model methods and tests:
  - random
  - translations_for_languages (associated to #81 above)

### Have Tests Been Added?
- [ ] No.
- [ ] Yes, but all tests are not passing.
- [x] Yes, and all are passing.

### Issues:
* Closes #26 

### Tracking Consistency:
- [x] Added appropriate labels
- [x] Addressed any rubocop violations
- [x] Added comments on my pull request, particularly in hard-to-understand areas

### CODE REVIEW

File: app/graphql/types/query_type.rb
Comment (lkriffell) [inline app/graphql/types/query_type.rb:52]: Is context basically saying given these two ids, return me a random topic with the translations for these two languages? I'm having trouble understanding because it's not very explicit that any translation is being returned

File: app/graphql/types/topic_type.rb
Comment (lkriffell) [inline app/graphql/types/topic_type.rb:11]: Is this where the specific translations (based on the two ids) are being set?

File: app/graphql/types/query_type.rb
Comment (leahriffell) [reply app/graphql/types/query_type.rb:52]: Context is allowing me to access those params (language_ids) elsewhere outside of this query, which was being used in `translations` under topic_type. Maybe it's more clear if I actually fetch each translation here vs in the type and then pass that in context instead.

File: app/graphql/types/topic_type.rb
Comment (leahriffell) [reply app/graphql/types/topic_type.rb:11]: Yes, exactly. But I just pushed up a refactor that instead of setting here, sets in the query and adds to context. What do you think?

Comment (lkriffell) [reply app/graphql/types/topic_type.rb:11]: I think it's easier to read with the refactor but maybe more it was more scalable before? I'll merge this and if we need to change it later to DRY up code then let's do that

**Review by lkriffell — APPROVED** [review]

**Pull request merged**
